extensions/components/com_redcore/admin/language/da-DK/da-DK.com_redcore.ini

```diff
@@ -80,7 +56,56 @@ COM_REDCORE_TRANSLATIONS_COMPONENT_LIST_TITLE="Komponentliste"
 COM_REDCORE_TRANSLATIONS_COMPONENT_LIST_DESC="Liste over alle installerede komponenter på dette site. Antallet i parenteser angiver, hvor mange indholdselementer er installeret for denne komponent."
 COM_REDCORE_TRANSLATIONS_SELECT_COMPONENT="Vælg komponent"
 COM_REDCORE_TRANSLATIONS_SELECT_CONTENT_ELEMENT="Vælg indholdselement"
-COM_REDCORE_CONFIG_TRANSLATIONS_LANGUAGE_FILTER="Joomla Plugin - Sprogfilter status:"
+COM_REDCORE_TRANSLATIONS_LANGUAGE_FILTER="Joomla Plugin - Sprogfilter status:"
+
+; Translation tables
+COM_REDCORE_TRANSLATION_TABLE_NAME="Tabelnavn"
+;COM_REDCORE_TRANSLATION_TABLE_NAME_DESC="Input table name without the prefix. Prefix #__ will be automatically added at the start of the table name"
```

Comment:
If we are not using these anymore maybe they should go to avoid confusing Transifex